rules/namingConventionRule.ts

```diff
@@ -98,13 +100,14 @@ enum Specifity {
     default = 1 << 9,
     variable = 2 << 9,
     function = 3 << 9,
-    parameter = 4 << 9,
-    member = 5 << 9,
-    property = 6 << 9,
+    functionVariable = 4 << 9,
```

Comment:
this can just be initialized with `Specifity.function` so you don't need to amend all following members.

Reply:
Oh, right, because `functionVariable` only needs to be greater than `variable` so that it takes precedence. And since a `function` cannot also be a `functionVariable`, they can have the same value. Is that correct?

Reply:
That's exactly how it works.